src/pages/gateway/upgrade.md

```diff
@@ -7,7 +7,17 @@ description: This page describes breaking changes and how to migrate from one ve
 
 This page lists breaking changes between versions of API Mesh for Adobe Developer App Builder. For a list of new features and resolved issues, refer to the [release notes](release-notes.md).
 
-To update to the latest version, run the following command:
+## Software updates
+
+If you are upgrading from a beta release to GA, you must update to `nvm` 18.x.x and update the mesh code.
```

Comment:
I think we need to include an aio upgrade command and for the plugin as well

Reply:
if we do an `aio update`, then it will update both aio and upgrade all the available plugins from the registry
hence npm install <plugin-name> is optional.
However, they could just do the `npm install -g @adobe/aio-cli-plugin-api-mesh` to get the latest version of just our plugin from the registry